src/Altinn.App.Core/Models/NationalIdentityNumberConverter.cs

```diff
@@ -0,0 +1,32 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Altinn.App.Core.Models;
+
+/// <summary>
+/// Json converter to transform between <see cref="string"/> and <see cref="NationalIdentityNumber"/>
+/// </summary>
+public class NationalIdentityNumberConverter : JsonConverter<NationalIdentityNumber>
```

Comment:
I see that some of these are internal and some are public. We should probably standardize, and also naming should either include `Json` or not

Reply:
I assume they have to be public to be used in a public class for de/serialization?

Reply:
As for naming, I had intended for them all to contain `json` in the name. Thoughts?

Reply:
Json in the name sounds good to me. Not sure if they need to be public, some of them are internal and I assume they work?